### PR TITLE
Acceleration timeline refactor

### PR DESCRIPTION
**Pull request by natsoni**
This PR re-does the previously reverted commits changes but without introducing the fee rates to the timeline, as requested: 

https://github.com/user-attachments/assets/a5ce394e-c96a-4c6a-91f9-5b10bb8ee5c8

### CODE REVIEW

File: frontend/src/app/components/acceleration-timeline/acceleration-timeline.component.ts [hunk 1]
```diff
@@ -19,15 +19,22 @@ export class AccelerationTimelineComponent implements OnInit, OnChanges {
...
+
+    window.setInterval(() => {
+      this.now = Math.floor(new Date().getTime() / 1000);
+      this.useAbsoluteTime = this.tx.status.block_time < this.now - 7 * 24 * 3600;
+    }, 60000);
   }
 
   ngOnChanges(changes): void {
-    this.now = Math.floor(new Date().getTime() / 1000);
     // Hide standard ETA while we don't have a proper standard ETA calculation, see https://github.com/mempool/mempool/issues/65
```
Comment (softsimon) [inline frontend/src/app/components/acceleration-timeline/acceleration-timeline.component.ts:38]: Maybe clean up all the commented code?

File: frontend/src/app/components/acceleration-timeline/acceleration-timeline.component.html [hunk 2]
```diff
@@ -1,6 +1,41 @@
...
+          <div class="acc-to-confirmed right go-faster"></div>
+        </div>
+        <div class="interval-spacer">
+        </div>
+        <div class="node" [id]="'confirmed'">
+          <div class="acc-to-confirmed left go-faster"></div>
+          <div class="shape-border waiting">
+            <div class="shape animate"></div>
+          </div>
+          <!-- <div>
```
Comment (softsimon) [inline frontend/src/app/components/acceleration-timeline/acceleration-timeline.component.html:31]: Cleanup commented code

File: frontend/src/app/components/acceleration-timeline/acceleration-timeline.component.ts [hunk 3]
```diff
@@ -19,15 +19,22 @@ export class AccelerationTimelineComponent implements OnInit, OnChanges {
...
+  useAbsoluteTime: boolean = false;
 
   constructor() {}
 
   ngOnInit(): void {
     this.acceleratedAt = this.tx.acceleratedAt ?? new Date().getTime() / 1000;
+    this.now = Math.floor(new Date().getTime() / 1000);
+    this.useAbsoluteTime = this.tx.status.block_time < this.now - 7 * 24 * 3600;
+
+    window.setInterval(() => {
```
Comment (softsimon) [inline frontend/src/app/components/acceleration-timeline/acceleration-timeline.component.ts:31]: This timeout continues after user leaves the view.
Set it to a variable and run `clearTimeout(this.timeout)`

**Review by softsimon — CHANGES_REQUESTED** [review]
See comments

File: frontend/src/app/components/acceleration-timeline/acceleration-timeline.component.ts (hunk 1 above)
Comment (natsoni) [reply frontend/src/app/components/acceleration-timeline/acceleration-timeline.component.ts:38]: Will probably need it to display the non-accelerated ETA later, once we improve the ETA calculation

**Review by softsimon — APPROVED** [review]
utACK

**Pull request merged**